src/test/java/dev/jbang/cli/TestRun.java

```diff
@@ -913,13 +913,13 @@ void testJavaAgentWithOptionParsing(@TempDir File output) throws IOException {
 
 		String result = run.generateCommandLine(main);
 
+		if (main.getArgs() != null) {
```

Comment:
this does not seem to test anything ? is this test generating a big command line ?

Reply:
Removed. Related to previous comment.